superchain/superchain.go

```diff
@@ -351,6 +386,34 @@ func (c ContractImplementations) Resolve(versions ContractVersions) (Implementat
 	if implementations.SystemConfig, err = resolve(c.SystemConfig, versions.SystemConfig); err != nil {
 		return implementations, fmt.Errorf("SystemConfig: %w", err)
 	}
+	// If the L2OutputOracle is not specified, we can assume that the L2OutputOracle is not used, and fault proofs are
+	// activated.
+	if implementations.L2OutputOracle, err = resolve(c.L2OutputOracle, versions.L2OutputOracle); err != nil {
```

Comment:
Wondering if we want to introduce a sentinel error for `resolve` and check it here specifically. Looks like `resolve` can have different reasons for error.

Reply:
Updated to check explicitly for an `ErrEmptyVersion` error